pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdSinks.java

```diff
@@ -168,6 +168,7 @@ public void runCmd() throws Exception {
             localRunArgs.add(new Gson().toJson(sinkConfig));
             for (Field field : this.getClass().getDeclaredFields()) {
                 if (field.getName().startsWith("DEPRECATED")) continue;
+                if(field.getName().startsWith("this$0")) continue;
```

Comment:
Since all anonymous classes and lambdas defined within a class will contain `$N` or `$lambdaN`, we should ignore everything with a `$`

Reply:
Done